file_paths.md

```diff
@@ -21,9 +21,12 @@ The files annotated by an '*' indicate that they are created by package installa
     /etc/puppetlabs *                                                    n/a
 
     /etc/puppetlabs/code *                # :codedir                     contents moved from :confdir
-        environments                      # :environmentpath
+        environments *                    # :environmentpath
+          production *
```

Comment:
If https://github.com/puppetlabs/puppet-agent/pull/86 gets merged, we should add `environment.conf` to this.